<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StreamDeck</class>
 <widget class="QMainWindow" name="StreamDeck">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Macropad</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	border:none;
	background-color: #003249;

}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QMainWindow {\n	background-color: #003249\n}</string>
   </property>
   <widget class="QPushButton" name="Button2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>110</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: #007ea7;
border: 1px solid white;
border-radius:10px;
color:#fff
}</string>
    </property>
    <property name="text">
     <string>Volume Down</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: #007ea7;
border: 1px solid white;
border-radius:10px;
color:#fff
}</string>
    </property>
    <property name="text">
     <string>Volume Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button5">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>230</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: #007ea7;
border: 1px solid white;
border-radius:10px;
color:#fff
}</string>
    </property>
    <property name="text">
     <string>Firefox</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: #007ea7;
border: 1px solid white;
border-radius:10px;
color:#fff
}</string>
    </property>
    <property name="text">
     <string>Take screenshot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>110</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: #007ea7;
border: 1px solid white;
border-radius:10px;
color:#fff
}</string>
    </property>
    <property name="text">
     <string>Mute/Unmute</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button6">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>230</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: #007ea7;
border: 1px solid white;
border-radius:10px;
color:#fff
}</string>
    </property>
    <property name="text">
     <string>YouTube</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>121</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
color: #fff;}
</string>
    </property>
    <property name="text">
     <string>Enter IP address:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectBtn">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>40</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: #007ea7;
color:#fff;
border-radius:7px;
 }</string>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
border-radius: 7px}</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>657</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
